### PR TITLE
Master bio

### PR DESCRIPTION
**Pull request by kkmattil**
Trinity modified, bs and seqtk added.

### CODE REVIEW

File: docs/apps/seqtk.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
+# Seqtk
+
+## Description
+
+Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. It seamlessly parses both FASTA and FASTQ files which can also be optionally compressed by gzip.
+
+
+llumina BaseSpace command line client,`bs`, can be used to retrievie data from the Illumina BaseSpace environment to Puhti.
```
Comment (Nortamo) [inline docs/apps/seqtk.md]: Is this by design here?, it's the exact same as in the bs.md file. Perhaps a link to the bs.md file would be better if we want to refer to it?

File: docs/apps/bs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## Available
+
+*    bs version 1.0.0 is available in Puhti
+
+## Usage
+
+The Illumina BaseSpace command line client starts with command:
+```text
+bs
+```
```
Comment (Nortamo) [inline docs/apps/bs.md]: The `bs` command is not in path. Does the user need to load/set/install something for these instructions to work?

File: docs/apps/seqtk.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+*    bs version 1.0.0 is available in Puhti
+
+## Usage
+
+
+Seqtk command syntax is:
+```text
+seqtk <command> <arguments>
+```
```
Comment (Nortamo) [inline docs/apps/seqtk.md]: The `seqtk` command is not in path. Does the user need to load/set/install something for these instructions to work?

File: docs/apps/bs.md (hunk 2 above)
Comment (kkmattil) [reply docs/apps/bs.md]: korjattu

File: docs/apps/seqtk.md (hunk 3 above)
Comment (kkmattil) [reply docs/apps/seqtk.md]: korjattu

File: docs/apps/trinity.md [hunk 4]
```diff
@@ -71,6 +79,8 @@ More information about running batch jobs, can be found from the chapter three o
 
 Please check the Trinity site to get hints for estimating the required resources,
 
+
+
  
 ## Using autoTrinotate in Taito
```
Comment (Nortamo) [inline docs/apps/trinity.md]: I would like to remove this Taito specific part, is that ok?

Comment (kkmattil) [reply docs/apps/trinity.md]: Haluaisin pitää näkyvillä maininnan siitä että autoTrinotate on saatavilla Taidossa.
Sitten kun ohjelma on myös puhdissa  niin  mainita voidaan poistaa. 

Tämä ei siis ole "näin käytät Trinityä Puhdissa" -ohje vaan  "Näin käytät Trinityä CSC:llä" -ohje.

**Comment by Nortamo** [comment]
Test fail due to manual anchors, I have a fix for this comming asap so it is ok.


Link #e1 in file ./docs/data/Allas/migration_tutorial.md on line 19 is broken

Link #e2 in file ./docs/data/Allas/migration_tutorial.md on line 20 is broken

The command "bash tests/check_broken_section_links.sh" exited with 1.

**Pull request merged**